### ISSUE TITLE
Add a light-mode/dark-mode toggle

### ISSUE THREAD

**Issue by sandbergja** [pulibrary/allsearch_frontend#443]
High priority

### User story

As a user, I want to try both the light mode and dark mode of this site to determine which is more usable for me, then use that one.

This came up in conversation with a Fable tester, who said:

> I think [dark mode] should be an opt-in.  Having it at the top.  It gives an option for the user, [...] because sometimes with their current assistive technology, they need to use dark mode with a specific website, but with other websites they don't have to.  So having the option to opt-in I think might be the best solution.

### Acceptance criteria

- [ ] I can toggle dark mode on or off without needing to change my browser or OS settings
- [ ] I have checked the toggle feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave

### Implementation notes

* [This article has a good approach](https://css-tricks.com/come-to-the-light-dark-side/), thanks to @stephanieleary for sharing it

**Comment by beth-german** [comment]
The UX/UI consultants asked why the search was so dark. Could we disable dark mode until a toggle option is available?
